packages/playground/src/api/functions/functionsApi.ts

```diff
@@ -0,0 +1,48 @@
+import { BaseResourceAPI, CDFHttpClient, MetadataMap } from '@cognite/sdk-core';
+import { IdEither, ItemsResponse } from '@cognite/sdk';
+import { Function, FunctionCreate, FunctionFilter } from '../../types';
+import { FunctionCallsApi } from './functionCallsApi';
+import { FunctionSchedules } from './functionSchedulesApi';
+
+export class FunctionsAPI extends BaseResourceAPI<Function> {
```

Comment:
wonder if you're missing a method: https://docs.cognite.com/api/playground/#operation/get-api-playground-projects-project-functions-function_name

Reply:
It is handled by retrieve, so I figured it is not needed.